src/server/lib/fetchTickerData.ts

```diff
@@ -40,9 +38,7 @@ const parse = (json: any): Promise<TickerData> => {
 
 const getTickerDataForTickerTypeFetcher =
     (stage: Stage, name: TickerName) => (): Promise<TickerData> => {
-        return fetch(tickerUrl(stage, name), {
-            timeout: 1000 * 20,
```

Comment:
Timeouts are done differently in the new api, but I don't think this is necessary here anyway.
We have a retry + polling mechanism for this